src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ParsingSimpleTypeModelBinder.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Runtime.ExceptionServices;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Binders
+{
+    /// <summary>
+    /// An <see cref="IModelBinder"/> for decimal, double, float, and their <see cref="Nullable{T}"/> wrappers.
+    /// </summary>
+    public class ParsingSimpleTypeModelBinder : IModelBinder
```

Comment:
`DecimalDoubleFloatModelBinderProvider` 😈 Hmm, maybe not.

Realized `decimal` is considered a "floating point" type,. So, `FloatingPointModelBinderProvider` or `FloatingPointTypeModelBinderProvider` work and I think I'll use the second.